wix-custom-labels/test/Service.withTags.service.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+/**
```

Comment:
The file name is a bit confusing, `Service.withTags.service.js`, and `Service.withTags.service.js`, maybe you can change this file to something more meaningful? `Service.withTwoTags.service.js`?

Reply:
done